pkg/controller/daemon/controller.go

```diff
@@ -316,8 +339,19 @@ func (dsc *DaemonSetsController) updateNode(old, cur interface{}) {
 		// A periodic relist will send update events for all known pods.
 		return
 	}
+	dsList, err := dsc.dsStore.List()
+	if err != nil {
+		glog.Errorf("Error enqueueing daemon sets: %v", err)
```

Comment:
Since updating a node can now error is there a need for this function to signal it's caller?